src/arrayMethodFilter.js

```diff
@@ -5,7 +5,15 @@
  */
 function applyCustomFilter() {
   [].__proto__.filter2 = function(callback) {
-    // write code here
+    const filteredArray = [];
+
+    for (let i = 0; i < this.length; i++) {
+      if (callback(this[i], i, this)) {
+        filteredArray.push(this[i]);
```

Comment:
можеш також пофіксити 
```suggestion
        filteredArray[filteredArray.length] = this[i];
```